hack/generate-operator-bundle-operatorhub.py

```diff
@@ -87,16 +87,32 @@ def generate_csv_base(version, prev_version, hive_image):
     if not os.path.exists(version_dir):
         os.mkdir(version_dir)
 
+    owned_crds = []
+
     # Copy all CSV files over to the bundle output dir:
     crd_files = os.listdir(crds_dir)
     for file_name in crd_files:
         full_path = os.path.join(crds_dir, file_name)
         if os.path.isfile(os.path.join(crds_dir, file_name)):
             shutil.copy(full_path, os.path.join(version_dir, file_name))
+            # Read the CRD yaml to add to owned CRDs list
+            with open(dest_path, 'r') as stream:
+                crd_csv = yaml.load(stream, Loader=yaml.SafeLoader)
+                owned_crds.append(
+                        {
+                            "description": crd_csv["spec"]["validation"]["openAPIV3Schema"]["description"],
```

Comment:
Should we stick with single quotes for consistency with the rest of this file?
```suggestion
                            'description': crd_csv['spec']['validation']['openAPIV3Schema']['description'],
```

Reply:
Passed e2e which I'm glad to see.

Yeah I'll update the quotes.